Aspire Budgeting/Views/Dashboard/DasboardCardsListView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  DasboardCardsListView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct DasboardCardsListView: View {
+
+  let cardViewItems: [DashboardCardView.DashboardCardItem]
+
+  var body: some View {
+    GeometryReader {g in
+      ScrollView {
+        VStack {
+          ForEach(0..<self.cardViewItems.count) { idx in
+            GeometryReader { geo in
+              BaseCardView<DashboardCardView>(
+//                       cardViewItem: self.cardViewItems[idx],
+                       minY: g.frame(in: .global).minY,
+                       curY: geo.frame(in: .global).minY,
+                baseColor: colorFor(idx: idx)) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/5d9f74b3b35e3fb9746b1333ffd07f18323321a5/Aspire Budgeting/Views/Dashboard/DasboardCardsListView.swift#L22"></span>Function parameters should be aligned vertically if they’re in multiple lines in a method call.<br />
<code>vertical_parameter_alignment_on_call</code> <code>DasboardCardsListView.swift:22</code></td>
    </tr>
  </tbody>
</table>